index.html

```diff
@@ -251,8 +255,12 @@ <h3> <span class="glyphicon glyphicon-wrench"></span> Efficacy </h3>
   	<div class="row">
     <div class="col-xs-12 col-sm-6 col-md-8" class="content">
     <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privacy Concerns</h3>
-    <p> There are serious legal questions surrounding cell site simulators. The manner in which these devices operate involves the indiscriminate data collection <a href="https://www.aclunc.org/blog/new-docs-doj-admits-stingrays-spy-innocent-bystanders">from phones of innocent bystanders</a>. </p>
-
+    <p>There are serious legal questions surrounding cell site simulators. The manner in which these devices operate involves the indiscriminate data collection <a href="https://www.aclunc.org/blog/new-docs-doj-admits-stingrays-spy-innocent-bystanders">from phones of innocent bystanders</a>. </p>
+	
+	<p>The largest concern surrounding cell site simulators is the deliberate secrecy around their use. The CPD does not keep a log of CSS use, and there are no directives in place outlining protocols for CSS related activity, as there are with most other surveillance technologies. The CPD has also signed a non-disclosure agreement (NDA) with the FBI requiring them to inform the bureau if there are any Freedom of Information Act (FOIA) requests from the public seeking information about this technology, enabling the FBI to <a href ="https://www.aclu-il.org/en/publications/stingrays-and-chicago-police-department#_ftn21">“seek to prevent disclosure through appropriate channels.”</a></p>
+	
+	<p>In 2017 the Citizen Privacy Protection Act (CPPA) went into effect in Illinois. This law requires any use of a CSS to be validated by a court order, and restricts CSS use to locating, tracking, and identifying communications devices, rather than capturing the content of communications. CPD officers <a href ="https://www.aclu-il.org/en/publications/stingrays-and-chicago-police-department#_ftn32">claim</a> that CSSs are almost never used without a court order, and that each order is carefully reviewed for compliance with the CPPA, but without adequate recordkeeping or avenues for the public to access those records, it is almost impossible to know whether this is the case.</p>
```

Comment:
I would expand on this a bit more. 

1. We don't know how CPD uses CSS because of the secrecy surrounding them. They claim to not keep a log of them being checked in / out. 
2. We know this because of numerous FOIAs filed by LPL and ACLU.
3. Numerous FOIAs for the court orders went unanswered.

Reply:
@b-meson 
I feel like (1) is already there?
```
<p>The largest concern surrounding cell site simulators is the deliberate secrecy around their use. The CPD does not keep a log of CSS use, and there are no directives in place outlining protocols for CSS related activity, as there are with most other surveillance technologies.
```
For (2) I can add something to the description:
```
<p>Cell site simulators (CSS) or IMSI catchers, also often referred to as Stingrays after the device sold by Harris Corporation, are controversial cellphone surveillance devices. They operate by masquerading as a cellphone tower, forcing any phones in the vicinity to connect to them. </p>
<p>Very little information about how CPD uses CSS is available to the public. Documents obtained through FOIA requests by LPL and ACLU have revealed that the CPD, FBI, and Harris Corporation have gone to great lengths to maintain secrecy around the purchase and use of CSS.
```
(3) I added here:
```
<p>In 2017 the Citizen Privacy Protection Act (CPPA) went into effect in Illinois. This law requires any use of a CSS to be validated by a court order, and restricts CSS use to locating, tracking, and identifying communications devices, rather than capturing the content of communications. CPD officers <a href ="https://www.aclu-il.org/en/publications/stingrays-and-chicago-police-department#_ftn32">claim</a> that CSSs are almost never used without a court order, and that each order is carefully reviewed for compliance with the CPPA. However, the CPD has<a href="https://www.aclu-il.org/en/publications/stingrays-and-chicago-police-department#_ftn31"> denied</a> numerous FOIA requests by the ACLU for court orders and warrants relating to CSS use. Without adequate recordkeeping or avenues for the public to access those records, it is impossible to tell whether CPD employs CSS lawfully and ethically.</p>
```

Reply:
2. `Documents obtained through FOIA lawsuits` 
3. LPL and ALCU both sent FOIAs we did for orders and warrants but we also sent requests to Cook County State's Attorney and the City Clerk.

Reply:
And the City Clerk and State's Attorney also denied the FOIA requests?